editor/editor_properties_array_dict.cpp

```diff
@@ -311,7 +311,7 @@ void EditorPropertyArray::update_property() {
 
 			size_slider = memnew(EditorSpinSlider);
 			size_slider->set_step(1);
-			size_slider->set_max(1000000);
+			size_slider->set_max(DBL_MAX);
```

Comment:
Yea I was doing more diving into the codebase and was looking at where it was actually used and it would be better to use a integer value instead. Would you still recommend INT_MAX over something like UINT32_MAX? I don't know a case where you would need a negative array size and there are a few spots where I see the size being cast to a unsigned 32 bit integer anyways.

Reply:
It's a signed integer so not `UINT32_MAX`, the `resize` function takes `int` so don't think it's safe